akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala

```diff
@@ -329,7 +329,8 @@ object MediaTypes extends ObjectRegistry[(String, String), MediaType] {
 
   val `application/atom+xml`                                                      = awoc("atom+xml", "atom")
   val `application/base64`                                                        = awoc("base64", "mm", "mme")
-  val `application/excel`                                                         = abin("excel", NotCompressible, "xl", "xla", "xlb", "xlc", "xld", "xlk", "xll", "xlm", "xls", "xlt", "xlv", "xlw")
+  @deprecated("This format is unofficial and should not be used", "10.1.6")
```

Comment:
I would add in the `deprecated` message the name of the equivalent one, I guess in this case is `application/vnd.ms-excel`